etc/kubernetes/helm/enterprise-gateway/templates/deployment.yaml

```
@@ -78,6 +78,14 @@ spec:
         nfs:
           server: {{ .Values.nfs.internalServerIPAddress }}
           path: "/usr/local/share/jupyter/kernels"
+{{- else if .Values.kernelpvc.enabled }}
```

Let's go with this slight name change...
```suggestion
{{- else if .Values.kernelspecs_pvc.enabled }}
```